src/platform/user/profile/vet360/reducers/index.js

```diff
@@ -228,10 +228,10 @@ export default function vet360(state = initialState, action) {
       return {
         ...state,
         addressValidation: {
-          ...state.addressValidation,
+          ...initialAddressValidationState,
           addressValidationError: action.addressValidationError,
           addressValidationType: action.addressValidationType,
-          validationKey: action.validationKey,
+          validationKey: action.validationKey || null,
```

Comment:
I saw an instance where the address validation error action didn't pass along the validationKey; in that case we'll want to set it to `null` rather than `undefined`